contracts/lido/ObserversArray.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.10;
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {ERC165Checker} from "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+import {IObserversArray} from "./interfaces/IObserversArray.sol";
+
+/// @author kovalgek
+/// @notice Manage observers.
+contract ObserversArray is Ownable, IObserversArray {
```

Comment:
Just few thoughts out loud regarding `Ownable` and moving out of `ObserversArray` functionality into a separate contract.
If we check for permissions in `ObserversArray` like this we kind of limit the ACL model used in the predecessor. 

This entangles `ObserversArray` with the additional functionality and makes it less reusable.
I'd suggest to either:
1) don't separate `ObserversArray` to contract - just include the code in `TokenRateNotifier`.  
2) move Ownable stuff to `TokenRateNotifier` and make `_addObserver` and `_removeObserver` internal

Reply:
I agree that we don't need a universal solution with observers here, but I didn't get why I should make _addObserver and _removeObserver internal.

Reply:
not valid anymore